it need python-xdg which we have only for 3.5 currently

diff --git a/src/api/python/speechd_config/spd-conf b/src/api/python/speechd_config/spd-conf
index 0e0a235..8aa88bb 100755
--- a/src/api/python/speechd_config/spd-conf
+++ b/src/api/python/speechd_config/spd-conf
@@ -1,4 +1,4 @@
-#!/usr/bin/env python3
+#!/usr/bin/python3.5
 
 # Helper script to be put in /usr/bin/ or a similar location
 # calling the appropriate python tool
